src/views/PopupView/InsertLabelNamesPopup/InsertLabelNamesPopup.tsx

```diff
@@ -1,30 +1,31 @@
-import {updateActivePopupType, updatePerClassColorationStatus} from '../../../store/general/actionCreators';
-import {AppState} from '../../../store';
-import {connect} from 'react-redux';
-import Scrollbars from 'react-custom-scrollbars';
-import {ImageButton} from '../../Common/ImageButton/ImageButton';
-import {LabelName} from '../../../store/labels/types';
-import {LabelUtil} from '../../../utils/LabelUtil';
-import {LabelsSelector} from '../../../store/selectors/LabelsSelector';
-import {LabelActions} from '../../../logic/actions/LabelActions';
-import {ColorSelectorView} from './ColorSelectorView/ColorSelectorView';
-import TextField from '@material-ui/core/TextField';
-import {Settings} from '../../../settings/Settings';
-import {withStyles} from '@material-ui/core';
-import {reject, sample, filter, uniq} from 'lodash';
-import {ProjectType} from '../../../data/enums/ProjectType';
-import {submitNewNotification} from '../../../store/notifications/actionCreators';
-import {INotification} from '../../../store/notifications/types';
-import {NotificationUtil} from '../../../utils/NotificationUtil';
-import {NotificationsDataMap} from '../../../data/info/NotificationsData';
-import {Notification} from '../../../data/enums/Notification';
+import React, { useState } from 'react';
+import './InsertLabelNamesPopup.scss';
+import { GenericYesNoPopup } from '../GenericYesNoPopup/GenericYesNoPopup';
+import { PopupWindowType } from '../../../data/enums/PopupWindowType';
+import { updateLabelNames } from '../../../store/labels/actionCreators';
+import { updateActivePopupType, updatePerClassColorationStatus } from '../../../store/general/actionCreators';
+import { AppState } from '../../../store';
+import { connect } from 'react-redux';
+import Scrollbars from 'react-custom-scrollbars-2';
+import { ImageButton } from '../../Common/ImageButton/ImageButton';
+import { LabelName } from '../../../store/labels/types';
+import { LabelUtil } from '../../../utils/LabelUtil';
+import { LabelsSelector } from '../../../store/selectors/LabelsSelector';
+import { LabelActions } from '../../../logic/actions/LabelActions';
+import { ColorSelectorView } from './ColorSelectorView/ColorSelectorView';
+import { Settings } from '../../../settings/Settings';
+import { reject, sample, filter, uniq } from 'lodash';
+import { ProjectType } from '../../../data/enums/ProjectType';
+import { submitNewNotification } from '../../../store/notifications/actionCreators';
+import { INotification } from '../../../store/notifications/types';
+import { NotificationUtil } from '../../../utils/NotificationUtil';
+import { NotificationsDataMap } from '../../../data/info/NotificationsData';
+import { Notification } from '../../../data/enums/Notification';
 
-const StyledTextField = withStyles({
+import { TextField } from '@mui/material';
+import { styled } from '@mui/system';
+
+const StyledTextField = styled(TextField)(({ theme }) => ({
```

Comment:
This is not working as expected, the style of input is changed. We would need to find a way to use new API of MUI, but at the same time don change the UI itself.

Reply:
my first take is to change the old ui and compile it without error. didn't thought that wasn't working.

Reply:
Sure I understand, let's just make sure it works as before ;)